src/main/java/com/iab/openrtb/request/BidRequest.java

```diff
@@ -32,6 +33,7 @@
      * Array of Imp objects (Section 3.2.4) representing the impressions
      * offered. At least 1 Imp object is required.
      */
+    @Wither
```

Comment:
Pls remove it. It is not good to have two approaches for one action. For this one second approach is for specific case